tcpmux.c

```diff
@@ -331,8 +331,16 @@ static int
 process_data(struct tmux_stream *stream, uint32_t length, uint16_t flags, 
 				void (*fn)(uint8_t *, int, void *), void *param)
 {
+    if(!stream){
+        return 0;
+    }
+	uint32_t id = stream->id;
+
 	if (!process_flags(flags, stream)) return 0;
 
+	if(!get_stream_by_id(id)){
```

Comment:
why need to check it again? in what condition if stream is not NULL but still cannot find it by its id?

Reply:
`process_flags()` may call `del_proxy_client_by_stream_id()`
https://github.com/liudf0716/xfrpc/blob/39b5d81231b01fae55359fb25d80ba91e385df72/tcpmux.c#L267

and `del_proxy_client` will free underlying stream struct, then the stream points to an invalid address.
https://github.com/liudf0716/xfrpc/blob/39b5d81231b01fae55359fb25d80ba91e385df72/client.c#L184